webapp/src/main/resources/locale/portlet/social/AdminsPortlet_en.properties

```diff
@@ -0,0 +1,14 @@
+social.admins.label=Admins
```

Comment:
1/ rename file to `PlatformAdminsWidget` (same as newly suggested name of Portlet)
2/ Move file under `locale.portlet` instead of `locale.portlet.social` (More standard location of I18N Files)
3/ Add File Declaration inside [crowdin.yml](https://github.com/Meeds-io/social/blob/develop/crowdin.yml#L561-L577) in order to allow synchronizing it with crowdin